CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## 0.24.0
 
-
+ * Added support to configure the worker thread pool size
```

Comment:
I would add a little bit more of context on how this pool is used. Just a few more info as in the section where you describe the corresponding env var.

Reply:
Wouldn't it just confuse users who until today had absolutely no clue there is some worker pool? Are you ready to answer the questions how should it be configured etc.?

Reply:
So I guess you are suggesting to remove it? I would agree on it, but if we leave the description we should be more precise on what "worker thread pool" is.

Reply:
I was thinking about it when doing my review. I do not have problem with it being there like this. My issue is that the CHANGELOG needs to be short and I'm afraid that by adding more details you will not be able to explain it and will just confuse people into thinking it is something important they should care about. Just my personal view of course.

Reply:
Tbh as it is now, I would remove it because it doesn't explain nothing clearly. It would raise more questions.

Reply:
how about `Support for configuring cluster-operator's worker thread pool size that is used for various sync and async tasks`?

Reply:
I am fine with that one, let's see @scholzj before making the change.

Reply:
Just approve it so that we can merge it ... ;-)